client/src/components/survey/SurveyPage4.jsx

```diff
@@ -61,16 +61,9 @@ const useStyles = makeStyles(theme => ({
       backgroundColor: `linear-gradient(45deg, #4760ff, #82a4f8)`,
     },
   },
-  chartDiv: {
-    display: 'flex',
-    marginTop: 50,
-    justifyContent: 'flex-end',
-  },
   innerChartDiv: {
     height: 700,
-  },
-  tagLine: {
-    marginTop: '40px',
+    overflow: 'auto',
```

Comment:
I'm just curious why we added this - I'm sure it's fine, but I always get extra cautious about `overflow` so want to double check

Reply:
I added it because otherwise the Fever Chart was getting overlapped with Behavioral Chart and vice versa. I tried playing around with styling till overflow worked.